great_expectations/expectations/window.py

```diff
@@ -0,0 +1,13 @@
+from great_expectations.compatibility import pydantic
+
+
+class PercentOffset(pydantic.BaseModel):
```

Comment:
Maybe name this ProportionalOffset instead of percentage if you expect a ratio.

Reply:
I'm leaning towards 'Offset' and its corresponding key to 'offset', so that we don't narrow ourselves in so early in this feature lifecycle

Reply:
cc @shiplet lets do a design review

Reply:
outcome: to change key to `offset`

Reply:
renamed key in 9e413e1e33eda30a28116d978f0869e11338f1c9

Reply:
I know the impl was not exactly your rec, but I think its an improvement - thoughts @billdirks ?

Reply:
Yes, this is great, thanks!